packages/common/src/ui/components/inputs/TextInput/BufferedTextInput.tsx

```diff
@@ -12,6 +12,7 @@ export type BufferedTextInputProps = BasicTextInputProps;
  */
 
 export const BufferedTextInput: FC<BufferedTextInputProps> = ({
+  InputProps,
```

Comment:
just so that I could change the background colour - needs to be white for the sidepanel inputs